### DISCUSSION TITLE
MultipleFileUploadEditor bug

### DISCUSSION THREAD

**Discussion by s10730047** [serenity-is/Serenity#7185] [General]
When using MultipleFileUploadEditor, I found a bug. 
EX: When I select three files at once, only two files are actually uploaded. 
When selecting multiple files, they never all upload together
How can I fix this?

Serenity version 8.4.4

**Comment by VictorTomaili** [comment]
Maybe you can update your serenity version?

https://github.com/serenity-is/Serenity/blob/master/CHANGELOG.md#850-2024-05-16
